ui/src/main/webapp/src/app/components/project-list/no-projects-welcome.component.html

```diff
@@ -7,7 +7,9 @@
     </div>
 
     <div class="welcome-help-text">
-        To learn more, visit the <a href="https://github.com/windup/windup-web/blob/master/README.md">
-            Red Hat Application Migration Toolkit Web Console Documentation</a>.
+        <template i18n="1st part">To learn more, visit the</template>
```

Comment:
@d-s - Exactly. 

I feel like maybe there was just some miscommunication and misreading of things in this discussion thread.